lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2460,6 +2460,67 @@ definitions:
           - 'NuRAN Cavium OC-LTE'
           - 'FreedomFi One'
         x-nullable: false
+      mme_pool_1:
+        type: string
+        format: ipv4
+        example: ''
+      mme_pool_2:
+        type: string
+        format: ipv4
+        example: ''
+      managementServer:
+        $ref: '#/definitions/managementServer'
+      sync_1588:
+        $ref: '#/definitions/sync_1588'
+  sync_1588:
+    description: sync_1588 Configuration for  eNodeb devices.
+    type: object
+    properties:
+      sync_1588_switch:
+        type: boolean
+        example: false
+      sync_1588_domain:
+        type: integer
+        format: int32
+        example: 0
+      sync_1588_msg_interval:
+        type: integer
+        format: int32
+        example: 60
+      sync_1588_delay_rq_msg_interval:
+        type: integer
+        format: int32
+        example: 60
```

Comment:
Is this interval in seconds? Add a description to specify units, and give a clear definition.